docs/reference/monitoring.rst

```diff
@@ -4,6 +4,9 @@ Monitoring
 The bundle ships with built-in health checks to be used by the `LiipMonitorBundle <https://github.com/liip/LiipMonitorBundle>`_ ,
 see the ``sonata.notification.backend.heath_check`` service.
 
+Before use, you need to install `HttplugBundle <http://docs.php-http.org/en/latest/integrations/symfony-bundle.html#installation>`_
+and configure it for use with any `HttpClient <https://packagist.org/providers/php-http/client-implementation>`_
```

Comment:
This makes your PR major

Reply:
Maybe you could instead make this PR add the possibility to use httplug, and then make a major PR to master, that would remove the dependency?

Reply:
You're right. This text section should be added in major PR.

Reply:
Just to be clear: it's not only about this piece of text. If I have guzzle installed and I upgrade SonataNotificationBundle to the next minor and get your changes, I should be able to get what I was getting before without any changes.

Reply:
Yes, you can get before without any changes. 
This [code](https://github.com/sonata-project/SonataNotificationBundle/pull/336/files#diff-22f21061edcb058dcf6be0c47c09b360R426) check if enabled `HttplgBundle`, then used php-http provider. Default is BC guzzle provider.

After merging code with master, I can open major PR (update docs and remove BC guzzle provider)